lib/srv/exec.go

```diff
@@ -137,9 +137,11 @@ func prepInteractiveCommand(ctx *ctx) (*exec.Cmd, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	// this configures shell to run in 'login' mode
+	// this configures shell to run in 'login' mode. from the bash man page:
```

Comment:
Good catch, rolled back this change, looks OpenSSH sets the first argument to `-{shell name}`: https://github.com/openssh/openssh-portable/blob/master/session.c#L1650-L1663